tests/test_client.py

```diff
@@ -682,13 +681,6 @@ def test_password_kwarg_works_for_password_auth(self):
         # Straightforward / duplicate of earlier basic password test.
         self._test_connection(password="pygmalion")
 
-    # TODO: more granular exception pending #387; should be signaling "no auth
-    # methods available" because no key and no password
-    @raises(SSHException)
```

Comment:
This can be accomplished in a reasonable way without `pytest_relaxed.raises`, look for examples in this file using `self.assertRaises()`